Styles/cart.css

```diff
@@ -24,11 +42,228 @@ button {
+        "nav nav"
+        "cart-content price-dialog";
+    grid-column-gap: 1em;
 }
 
 .cart-heading {
-    margin-bottom: 0;
-    margin-top: 1.5em;
+    grid-area: nav;
+    margin: 1.5em auto;
     text-align: center;
     text-transform: uppercase;
 }
+
+.cart-container {
+    grid-area: cart-content;
+    margin-bottom: 2em;
+
+    /* grid properties */
+    display: grid;
+    justify-content: stretch;
+    grid-row-gap: 1em;
+}
+
+.card-item {
+    margin: 0 auto;
+    width: 95%;
+    max-width: 40em;
+}
+
+.card {
+    gap: 0.5em;
+}
+
+.card-horizontal .card-image {
+    width: 8rem;
+    border-radius: 0.3em;
+    margin: 0.5em;
+    margin-bottom: 0;
+    margin-right: 0;
+    border: 1px solid #808080;
+}
+
+.card-horizontal .card-header-container {
+    margin: 0;
+    margin-top: 0.5em;
+}
+
+.card-horizontal .card-heading {
+    font-size: 1.2rem;
+    font-weight: 600;
+    padding: 0 1.2rem;
+}
+
+.card-horizontal .card-subheading {
+    margin: 0;
+    padding: 0 1.2rem;
+    margin-top: 0.3rem;
+    font-size: 1rem;
+
+    /* flex properties */
+    display: flex;
+    align-items: center;
+    gap: 0.5em;
+}
+
+.discount-price {
+    font-size: 1.1rem;
+    font-weight: 500;
+}
+
+.rating-svg-icons {
+    cursor: default;
+}
+
+.mrp {
+    text-decoration: line-through;
+    color: #808080;
+}
+
+.quantity-display {
+    font-size: 0.8rem;
+    font-weight: 500;
+    border: 2px solid #808080;
+    border-radius: 0.3em;
+    padding: 0em 0.5em;
+}
+
+.quantity-button {
+    /* flex properties */
+    display: flex;
+    align-items: center;
+}
+
+.quantity-icons {
+    font-size: 1.1rem !important;
+    color: #808080;
+}
+
+.card-horizontal .card-footer {
+    padding: 0;
+}
+
+.card-primary-buttons-container {
+    width: 100%;
+    border-top: 1px solid #808080;
+}
+
+.card-primary-buttons-container .card-button {
+    width: 65%;
+    background-color: transparent;
+    border-radius: 0;
+    color: #808080;
+    font-size: 0.9rem;
+    font-weight: 500;
+    padding: 0.5em 1em;
+    font-family: "Roboto", sans-serif;
+
+    /* flex properties */
+    display: flex;
+    justify-content: center;
+}
+
+.card-primary-buttons-container .card-button:nth-child(1) {
+    width: 35%;
+    border-right: 1px solid #808080;
+}
+
+.price-modal {
+    grid-area: price-dialog;
+    justify-self: start;
+}
+
+.modal {
+    max-width: 95%;
+}
+
+.modal-title {
+    font-weight: 600;
+    font-size: 1.3rem;
+}
+
+.confirmation-modal .modal-body {
+    align-items: unset;
+}
+
+.confirmation-modal .modal-body .modal-text {
+    width: max-content;
+    padding-left: 0;
+    cursor: default;
+}
+
+.modal-text:hover {
+    background-color: transparent;
+}
+
+.money-details {
+    /* flex properties */
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.highlight-item {
+    color: var(--secondary-color);
+}
+
+.divider-line {
+    width: 100%;
+    color: #ececec;
+    margin: 0;
```

Comment:
use variabe

Reply:
I have asked vikram this query in the thread, would do the changes after the response. Thanks!